src/wazuh_db/wdb_parser.c

```diff
@@ -652,7 +652,7 @@ int wdb_parse(char * input, char * output, int peer) {
 
         mdebug2("Global query: %s", query);
 
-        if (wdb = wdb_open_global(), !wdb) {
+        if (wdb = wdb_open_global(), !wdb || !wdb->enabled) {
```

Comment:
Do you think that we need a different message in case of `wdb_global->enabled == FALSE ` ? Maybe `Couldn't open DB global` sounds like an error, and in fact is a special situation.

Reply:
As we discussed with team, we shouldn't allow the whole upgrade process if the database isn't enabled.

Reply:
Fixed